### PR TITLE
Corrections and clarity

### PR DESCRIPTION
**Pull request by losowsky**
Fixed the incorrect thresholds and made the description clearer

### CODE REVIEW

File: docs/source/03-07-product-guide-trust.md [hunk 1]
```diff
@@ -3,47 +3,53 @@ title: Trust
...
 
 Here are the default thresholds:
 
 ```text
--2 and lower: Unreliable
--1 to +2: Neutral
-+3 and higher: Reliable
+-1 and lower: Unreliable
+0 to +1: Neutral
++1 and higher: Reliable (we don't do anything with this label right now)
```
Comment (wyattjoh) [inline docs/source/03-07-product-guide-trust.md]: This should actually be +2:

https://github.com/coralproject/talk/blob/4413e5616c8655b30f08144e78606033dd4a62a5/config.js#L245

**Review by wyattjoh — DISMISSED** [review]
I think there's only one change.

File: docs/source/03-07-product-guide-trust.md (hunk 1 above)
Comment (losowsky) [reply docs/source/03-07-product-guide-trust.md]: oh, good point.

**Review by wyattjoh — APPROVED** [review]
Looks good to me!

**Pull request merged**